### DISCUSSION TITLE
Remove /callback from AvailableCommandsPacket

### DISCUSSION THREAD

**Discussion by RealBauHD** [PaperMC/Paper#10840] [Ideas]
### Is your feature request related to a problem?

No it is not related to any problem.

### Describe the solution you'd like.

Like [Velocity](https://github.com/PaperMC/Velocity/blob/46cf9be297e366bbab2a2941d35246a85d1591e9/proxy/src/main/java/com/velocitypowered/proxy/connection/backend/BackendPlaySessionHandler.java#L347) remove the /callback command from the AvailableCommandsPacket so it is not sent, as it is only used for internal processes and should not be executed by players themselves.

### Describe alternatives you've considered.

Alternatively, the command could be removed by the ``PlayerCommandSendEvent``.

**Comment by Machine-Maker** [comment]
This sounds like a good idea to handle on our end.
